tests/data/test_make_pv_data.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import xarray as xr
+from datetime import datetime
+
+from quartz_solar_forecast.pydantic_models import PVSite
+
+def mock_enphase_data():
+    data = pd.DataFrame({
+        'timestamp': [
+            datetime(2024, 6, 5, 11, 25),
+            datetime(2024, 6, 5, 11, 30),
+            datetime(2024, 6, 5, 11, 35)
+        ],
+        'power_kw': [0.5, 0.6, 0.7]
+    })
+    return data
+
+def test_make_pv_data_with_enphase(mock_enphase_data):
```

Comment:
Great that you added a test for this, I think it needs a bit of updating, ideally you want to avoid duplicating code you have written before in the `make_pv_data` function so you want to import that function here and actually use it in your test, however you don't want to be grabbing data via enphase so you want to mock what the `get_enphase_data` call inside `make_pv_data` actually returns, in this case it would be the mock_enphase_data you have created above, an example of how to do this is here https://dev.to/quame_jnr1/mocking-requests-in-python-2anc